lib/MetaCPAN/Sitemap.pm

```diff
@@ -58,7 +58,7 @@ sub process {
     # XXX Remove this hardcoded URL
     my $es = Search::Elasticsearch->new(
         cxn_pool         => 'Static::NoPing',
-        nodes            => ['api.metacpan.org'],
+        nodes            => ['api-v1.metacpan.org'],
```

Comment:
This should work: https://github.com/CPAN-API/metacpan-web/blob/master/app.psgi#L58